utils/data_preprocessing.py

```diff
@@ -201,9 +203,15 @@ def load_data(dir: str, unwanted_keywords: Set[str]):
 
 def load_data_from_frontend(docs: dict):
```

Comment:
I think loading the content from the data that is send from the front end is a bad idea, because everywhere else in the back end we use SolrDocs as representation of a document and here you have a dict. In apply_tagging_method you have to load the docs from solr anyways after create_automated_keywords because you need to update them.

Reply:
I am actually also thinking whether we should remove the content field from the frontend. It should't be there.

Reply:
Perhaps a `/documents` route which sends documents without content and a `/documents/<id>` route which sends a specific document with content?